src/styles/index.css

```diff
@@ -0,0 +1,261 @@
+@charset "UTF-8";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.header__box {
+  display: flex;
+  height: 60px;
+  flex-grow: 1;
+}
+
+.header__box-logo {
+  position: absolute;
+  z-index: 1;
+  top: 10px;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.header__container {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  margin-left: 0;
+  margin-right: 50px;
+}
+
+.header__container-titles {
+  list-style: none;
+  align-items: center;
+  display: flex;
+}
+
+.menu {
+  color: #060b35;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  transition: all 300ms ease;
+}
+
+.menu:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  display: inline-block;
+  margin-left: 0;
+  height: 60px;
+  line-height: 60px;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  transition: background-color 0.3s;
+  border-radius: 10px;
+  border-style: 10px solid;
```

Comment:
The `border-style` property is incorrectly used here. It should be `border` or `border-width` to define the border's width, style, and color. Consider removing it or using `border: 10px solid transparent;` if a border is intended.